README.md

```diff
@@ -57,37 +57,16 @@ To get the source code from GitHub, type
     cd ./azure-sdk-for-php
 
 > **Note**
-> 
+>
 > The PHP Client Libraries for Microsoft Azure have a dependency on the [HTTP_Request2](http://pear.php.net/package/HTTP_Request2), [Mail_mime](http://pear.php.net/package/Mail_mime), and [Mail_mimeDecode](http://pear.php.net/package/Mail_mimeDecode) PEAR packages. The recommended way to resolve these dependencies is to install them using the [PEAR package manager](http://pear.php.net/manual/en/installation.php).
 
-
 ##Install via Composer
+1. Make sure you have installed **[Compoeser](https://getcomposer.org/doc/00-intro.md#introduction)** in your system.
+It can be installed locally or globally depending on your own needs.
 
-1. Create a file named **composer.json** in the root of your project and add the following code to it:
```

Comment:
Short recap:
```
composer create-project microsoft/windowsazure /path/to/install/dir ^0.4@dev
```
will install the standalone package and its dependencies, just like `git clone https://github.com/Azure/azure-sdk-for-php.git /path/to/install/dir && composer install`
Obviously, that is useful when the user wants to execute test or needs to check some features of the library outside any other project.
On the other hand, the removed lines aims to users to manually edit its `composer.json` file. Even for beginners it should be a odd case, since if they are trying to use Composer as dependency manager, they should have other declarations specific to its own project, like the name, any other dependencies or autoload mechanisms.
In that case, a better explanation could be:
```markdown
In order to install this library with Composer as a dependency of your project, you must run the following command:

`php composer require microsoft/windowsazure:^0.4@dev`

which will add the following requirement to your `composer.json` file:
```json
{
    "require": {
        "microsoft/windowsazure": "^0.4@dev"
    }
}
```
Please, note that this command will create a `composer.json` file if it doesn't exist previously, so that is no issues in both situations.